src/core/physics/CollisionInfo.js

```diff
@@ -0,0 +1,70 @@
+
+define(["lib/classy",
+        "lib/three"
+    ], function(Class, THREE) {
+
+/**
+    Info object that is returned when collision happens between two rigidbodies.
+    @class CollisionInfo
+    @constructor
+    
+    @param {Entity|Null} bodyA
+    @param {Entity|Null} bodyB
+    @param {THREE.Vector3} position
+    @param {THREE.Vector3} normal
+    @param {Number} distance
+    @param {Number} impulse
+    @param {boolean} newCollision
+*/
+var CollisionInfo = Class.$extend(
```

Comment:
I think you could remove the getters as they don't do anything :) I doubt the plan is to add anything special inside the getters in the future.

I like that its a separate, clear, documented object that can be new'd up and reused by any other code. But just using the properties directly would be a more js way of doing it :) (just an thought)